### ISSUE TITLE
"M117 L180 S8640" fails to roundtrip

### ISSUE THREAD

**Issue by niteria** [hackerspace/gcodehs#1]
Hi,

First of all, thank you for the library. I've recently used it to build a little script that strips all the filament extrusion from the gcode for my 3d printer to reproduce noisy movement.

I've run into a tiny issue. Here's a small reproducer:
```
import Data.Attoparsec.ByteString (parseOnly)
import Data.GCode.Parse
import Data.GCode.Pretty
import Data.String (fromString)

main = do
  let Right gcode = parseOnly parseGCode (fromString "M117 L180 S8640\n")
  putStrLn $ ppGCode gcode
```

Output:
```
Repro: src/Data/GCode/Pretty.hs:(83,1)-(89,21): Non-exhaustive patterns in function axisColor
```

For reference, M117 means Set LCD Message on my 3d printer. http://marlinfw.org/docs/gcode/M117.html

Thanks,
Bartosz

**Comment by niteria** [comment]
Tagging @sorki, in case the notification went to unwatched account.

**Comment by sorki** [comment]
Sorry this took so long, I've fixed it two years ago, started adding more stuff and then I've had to leave it and totally forgot about it. It now has a testsuite as well as bunch of more exciting features :) Will release a new version soon-ish.

**Comment by niteria** [comment]
No worries, thanks for the fix!

**Comment by sorki** [comment]
Released `0.1.2.0` to Hackage just now - https://hackage.haskell.org/package/gcodehs-0.1.2.0/changelog

Should be good from now on ;)
